assets/js/custom/Index.js

```diff
@@ -29,10 +29,11 @@ function Index (clickStats, homepageClickStats, confirmButtonText) {
       let url = baseUrl + '/api/projects/?category=' + category
 
       /* eslint-disable no-undef */
-      if (flavor !== undefined && flavor !== 'pocketcode') {
+      if ((flavor !== undefined && flavor !== 'pocketcode') || category === 'example') {
```

Comment:
The flavor should not be undefined, not even when the category is "example". Modify the parentheses.